### PR TITLE
Concrete and mortar tweaks

### PR DESCRIPTION
**Pull request by Rail-Runner**
1. Make mortar and concrete spawn in a bag from which they can be removed, and change their names, weight and descrpitions to reflect that.
2. Remove canvas bag from mortar and concrete recipe.
3. Replace rocks with pebbles in concrete recipe and allow using clean water instead of water for making it.

### CODE REVIEW

File: data/json/items/melee.json [hunk 1]
```diff
@@ -779,18 +779,21 @@
...
-        "name_plural": "bags of concrete",
-        "description": "A bag of concrete, ready to be used in a construction project.",
+        "name": "concrete",
+        "name_plural": "concrete",
+        "description": "Some concrete, ready to be used in a construction project.",
+        "container" : "bag_canvas",
         "price": 10000,
         "material": "stone",
-        "weight": 1000,
+        "weight": 585,
```
Comment (Coolthulhu) [inline data/json/items/melee.json]: Weight is per unit.
One unit is 4 volume.

Comment (Darkyhard) [inline data/json/items/melee.json]: Actually I've just removed weight of a bag from that, to keep weight consistent for spawned bags of concrete. How much should a concrete or mortar itself actually weigh?

Comment (Coolthulhu) [inline data/json/items/melee.json]: Roughly 4 times as much as it currently weights.

**Comment by Darkyhard** [comment]
Done.

File: data/json/recipes/recipe_others.json [hunk 2]
```diff
@@ -61,13 +61,11 @@
      [ "material_sand", 25]
      ],
      [
-     [ "rock", 20]
+     [ "pebble", 20]
```
Comment (RadaRadaRada) [inline data/json/recipes/recipe_others.json]: Only 20 pebbles? That translates into 2 rocks instead of 20.

Comment (Darkyhard) [inline data/json/recipes/recipe_others.json]: Mortar uses 20 pebbles to craft too, rocks aren't an option there.

**Pull request merged**